pkg/apis/infra/v1alpha1/cluster.go

```diff
@@ -69,7 +69,7 @@ type ControlPlaneConfig struct {
 }
 
 type CNIConfig struct {
-	// Type is the type of CNI (e.g. calico, canal, cilium, flannel, kube-ovn, kube-router, macvlan, weave).
+	// Type is the type of CNI. The supported types are: calico, canal, cilium.e.g. calico, canal, cilium, flannel, kube-ovn, kube-router, macvlan, weave
```

Comment:
remove ` e.g. calico, canal, cilium, flannel, kube-ovn, kube-router, macvlan, weave`

since these are not supported

Reply:
these are supported， it is a typo and has been updated